src/web/api/src/main/java/org/locationtech/geogig/web/api/CommandBuilder.java

```diff
@@ -269,6 +269,11 @@ static BranchWebOp buildBranch(ParameterSet options) {
         BranchWebOp command = new BranchWebOp();
         command.setList(Boolean.valueOf(options.getFirstValue("list", "false")));
         command.setRemotes(Boolean.valueOf(options.getFirstValue("remotes", "false")));
+        command.setBranchName(options.getFirstValue("branchName", null));
+        // TODO: enable these options if necessary
+        //command.setAutoCheckout(Boolean.valueOf(options.getFirstValue("autoCheckout", "false")));
```

Comment:
@groldan I've tested these, but commented them out for now. I don't know how much we want to open up through the web API. Some of these options could allow the repo to get messy if used carelessly.